pkg/operator2/oauth.go

```diff
@@ -150,10 +151,11 @@ func (c *authOperator) handleOAuthConfig(
 	}
 
 	cliConfigBytes := encodeOrDie(cliConfig)
+	newConsoleConfig := managedConsoleConfigBytes(managedConsoleConfig)
 
-	completeConfigBytes, err := resourcemerge.MergeProcessConfig(nil, cliConfigBytes, operatorConfig.Spec.UnsupportedConfigOverrides.Raw)
+	completeConfigBytes, err := resourcemerge.MergeProcessConfig(nil, cliConfigBytes, newConsoleConfig, operatorConfig.Spec.UnsupportedConfigOverrides.Raw)
```

Comment:
You cannot merge the branding config map on top of the OAuth server config.

Reply:
yea figured that-but wasn't sure how the overriding of console config was going to happen so wanted to run this to see (notice the WIP and /hold) : )